Sources/UI/Notifications/Services/InfluenceTracker/InfluenceTrackerService.swift

```diff
@@ -59,7 +59,7 @@ class InfluenceTrackerService: InfluenceTracker {
             var campaignID: ID
             var receivedAt: Date
             
-            var attributeValue: AttributeValue {
```

Comment:
`AttributeRepresentable` has been removed in this PR. You'll need to update this to match the approach you took in the `Beacon` and `Geofence` classes.